packages/commonwealth/client/scripts/state/api/user/updateUser.ts

```diff
@@ -40,27 +53,6 @@ const useUpdateUserMutation = ({
         }
       });
 
-      // if `userId` matches auth user's id, refetch profile-by-id query for auth user.
-      if (user.id === updated.id) {
-        const keys = [
-          [ApiEndpoints.FETCH_PROFILES_BY_ID, undefined],
-          [ApiEndpoints.FETCH_PROFILES_BY_ID, user.id],
-        ];
-        keys.map((key) => {
-          queryClient.cancelQueries(key).catch(console.error);
-          queryClient.refetchQueries(key).catch(console.error);
-        });
-
-        // if user profile has a defined name, then set welcome onboard step as complete
-        if (
-          updated.profile.name &&
-          updated.profile.name !== DEFAULT_NAME &&
-          !user.isWelcomeOnboardFlowComplete
-        ) {
-          user.setData({ isWelcomeOnboardFlowComplete: true });
-        }
```

Comment:
we should'nt remove this

Reply:
I put it on top... using the updated property coming from the backend